website/docs/docs/collaborate/explore-projects.md

```diff
@@ -57,9 +57,10 @@ To explore the lineage graphs of tests and macros, view [their resource details
     - Refocus on the node and its downstream nodes only
     - Refocus on the node and it upstream nodes only
     - View the node's [resource details](#view-resource-details) page
-- Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
-    - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
-    - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
+- Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details. In the side panel's upper right corner:
+    - Click the View Resource icon to [view the resource details](#view-resource-details). 
+    - Click the [Open in IDE](#open-in-ide) icon to examine the resource using the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).
```

Comment:
```suggestion
    - Click the **[Open in IDE](#open-in-ide) icon** to examine the resource using the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).
```

Reply:
thanks for flagging!

i looked at the style guide and ran a search in our git repo. it doesn't seem like we have bolding rules for icons and the prior art in our repo isn't consistent. i think this is non-blocking. should we bring this up in our team meeting tomorrow?